DataStore/interfaceview.xml

```diff
@@ -0,0 +1,283 @@
+<InterfaceView version="1.1" asn1file="datastore.asn">
```

Comment:
I think I am missing publish/subscribe interfaces. We can discuss this if needed.

Reply:
subscribe_to_event was implemented, but I have issues with running the project